flaskr/tweets.py

```diff
@@ -32,12 +32,13 @@ def index():
     return render_template('tweets/all_tweets.html', tweets_authors=zip(tweets, authors))
 
 
-@bp.route("search_word/<word>", methods=["GET"])
-def search_for_word(word):
+@bp.route("/search_word", methods=["GET"])
+def search_for_word():
     db = get_db()
     authors = list()
     tweets = list()
-    word = word.lower()
+    # Do we need to sanitize the input?
```

Comment:
I don't think that our clone will be the target of a lot of attacks, maybe we can sanitize everything for the second due date ?